torch/_subclasses/fake_tensor.py

```diff
@@ -1883,6 +1931,10 @@ def from_tensor(
             constraint_dims=constraint_dims,
             memoized_only=memoized_only,
         )
+        self.policy_cache[tensor] = FakificationPolicy(
+            source=source, ignore_subclass=ignore_subclass
+        )
```

Comment:
There needs to be some sort of way to make sure people don't add new arguments to this function without also updating policy if it is necessary. Maybe a comment on the kwargs list is enough